Code/Source/svFSI/ComMod.h

```diff
@@ -1448,6 +1453,9 @@ class ComMod {
     /// @brief Time step size
     double dt = 0.0;
 
+    /// @breif Time step size of the precomputed state-variables
```

Comment:
Very minor, but important for proper Doxygen: breif -> brief

Reply:
fixed spelling mistake in zasexton:precomputed-velocities-#202